lib/wise_homex/api_config.exs

```diff
@@ -1,6 +1,46 @@
 # The definition of all endpoints that are autogenerated in WiseHomex
 
 endpoints = [
+  %{
+    endpoints: [:index, :show, :create, :update, :delete],
+    model: WiseHomex.Account,
+    name_plural: "accounts",
+    name_singular: "account",
+    path: "/accounts",
+    type: "accounts"
+  },
+  %{
+    endpoints: [:index, :delete],
+    model: WiseHomex.AccountUser,
+    name_plural: "account_users",
+    name_singular: "account_user",
+    path: "/account-users",
+    type: "account-users"
+  },
+  %{
+    endpoints: [:index, :show, :create, :update],
+    model: WiseHomex.Address,
+    name_plural: "addresses",
+    name_singular: "address",
+    path: "/addresses",
+    type: "addresses"
+  },
+  %{
+    endpoints: [:index, :show, :delete],
+    model: WiseHomex.AdminIntegration,
+    name_plural: "admin_integrations",
+    name_singular: "admin_integration",
+    path: "/admin-integrations",
+    type: "admin-integrations"
+  },
+  %{
+    endpoints: [:create, :update],
+    model: WiseHomex.AdminIntegration,
+    name_plural: "admin_integrations_unik",
+    name_singular: "admin_integration_unik",
+    path: "/admin-integrations/unik",
+    type: "admin-integrations"
+  },
```

Comment:
With two endpoints with the same model, the thing in `ApiDefinition.type_to_model_mappings` should check for duplicate keys that does not have the same value.

Tip:

```elixir
iex(2)> [{:a, 1}, {:a, 2}] |> Enum.into(%{})      
%{a: 2}
```

Reply:
@lasseebert Ah, I see. Good point! :+1: Of course it could be an issue if we overwrite a type with another model. In this particular case it's not an issue, but it could be. We should `raise` in that case, right?

Reply:
Yes, I agree. It should just crash, since the data is then broken.

Reply:
I added a check for it in https://github.com/wise-home/wise_homex/pull/126/commits/30d77419af900849f471225da60f14abed2604f9 .. What do you think about it?

Reply:
Very nice :+1: